docs/interface.md

```diff
@@ -30,9 +30,9 @@ partial interface WorkerNavigator {
 };
 
 dictionary ShareData {
-  DOMString? title;
-  DOMString? text;
-  DOMString? url;
+  DOMString title;
+  DOMString text;
+  USVString url;
```

Comment:
Thanks for the explanation.

> but basically anything that will end up serialized (encoded) and sent over the network benefits

Well we don't know what's going to happen to these fields... they're going to experience all sorts of transformations. In Chrome's Web Share Target prototype they are going to be encoded as UTF-8 and put into a URL. They could also be passed to arbitrarily many native apps that could be written in any programming language. It isn't a good thing if they can contain in valid UTF-16 surrogates.

Could all of these be USVString (ech my fingers keep typing "USBString")? Or do I have to have a specific reason to use that type?